### PR TITLE
Add LaTeX support

### LINKED ISSUE #542: LaTeX support
**Is your feature request related to a problem? Please describe.**
Add LaTeX support

**Describe alternatives you've considered**
[Chktex](https://ctan.org/pkg/chktex?lang=en) seems to be a good candidate.

### PR DESCRIPTION
**Pull request by assignUser**
Closes #542 

I added Latex support via [ChkTex](https://www.nongnu.org/chktex/). I have not run all test yet (building the image takes a long time on my machine) but it felt pretty straight forward.

### Author/Contributor
- [x] If documentation is needed for this change, has that been included in this pull request

### Reviewing Maintainer 
- [ ] Label as `breaking` if this is a large fundamental change
- [ ] Label as either `automation`, `bug`, `documentation`, `enhancement`, `infrastructure`, or `performance`

### CODE REVIEW

**Comment by assignUser** [comment]
@admiralAwkbar everything should be in alphabetical order this time ;)
This could be extended to use chktex to also lint "cweb" files but I am not sure how much an interest exist for that.

File: Dockerfile [hunk 1]
```diff
@@ -248,6 +249,15 @@ RUN luarocks install luacheck
 COPY --from=lintr-lib /usr/lib/R/library/ /home/r-library
 RUN R -e "install.packages(list.dirs('/home/r-library',recursive = FALSE), repos = NULL, type = 'source')"
 
+##################
+# Install chktex #
+##################
+RUN wget --tries=5 http://mirrors.ctan.org/support/chktex.zip -O - -q | unzip -q -\
```
Comment (assignUser) [inline Dockerfile:255]: I don't understand why unzip is failing here but not here: https://github.com/github/super-linter/blob/61a2c4e12f5aee2b4428dec346a32e4ff12b7b77/Dockerfile#L205

File: docs/disabling-linters.md [hunk 2]
```diff
@@ -602,6 +603,48 @@ import package.b.*
...
 
+## LaTeX
+
+- [ChkTex](https://www.nongnu.org/chktex/)
+
+### ChkTex Config file
+
+- `.github/linters/chktexrc`
+- You can pass multiple rules and overwrite default rules
+- File should be located at: `.github/linters/.luacheckrc`
```
Comment (admiralAwkbar) [inline docs/disabling-linters.md:614]: I think the file name is wrong here?

File: lib/linter.sh [hunk 3]
```diff
@@ -69,6 +69,9 @@ JAVASCRIPT_LINTER_RULES="${DEFAULT_RULES_LOCATION}/${JAVASCRIPT_FILE_NAME}" # Pa
 JAVASCRIPT_STANDARD_LINTER_RULES=''                                         # ENV string to pass when running js standard
 # Default linter path
 LINTER_RULES_PATH="${LINTER_RULES_PATH:-.github/linters}" # Linter Path Directory
+# LaTeX Vars
+LATEX_FILE_NAME='chktexrc'                                   # Name of the file
+LATEX_LINTER_RULES="${DEFAULT_RULES_LOCATION}/${LATEX_FILE_NAME}" # Path to the Lua lint rules
```
Comment (admiralAwkbar) [inline lib/linter.sh:74]: comment seems not updated

**Comment by admiralAwkbar** [comment]
@assignUser I think 2 small updates and this is golden! thank you for the strong contributions!

**Comment by admiralAwkbar** [comment]
@assignUser 
```bash
File:[/tmp/lint/.automation/test/latex/latex_good_1.tex]
2020-08-19 13:29:21 [ERROR ]   Found errors in [chktex] linter!
2020-08-19 13:29:21 [ERROR ]   [chktex: WARNING -- Could not find global resource file.]
2020-08-19 13:29:21 [ERROR ]   Linter CMD:[chktex -g -q -l /tmp/lint/.github/linters/chktexrc /tmp/lint/.automation/test/latex/latex_good_1.tex]
```

**Comment by assignUser** [comment]
@admiralAwkbar first off: 😂 
image
Yeah chktex seems to ignore the `-g` flag, I adjusted the dockerfile so an empty config file in the default directory is created. That should fix it.

**Comment by admiralAwkbar** [comment]
@assignUser I was able to make it run by getting rid of the `-l` and then pointing with `-g /path/to/file` but not sure if that is an acceptable answer

**Comment by assignUser** [comment]
> @assignUser I was able to make it run by getting rid of the `-l` and then pointing with `-g /path/to/file` but not sure if that is an acceptable answer

thats weird as -g is supposed to be a boolean flag to turn of that warning..., I'll look into it

**Comment by assignUser** [comment]
@admiralAwkbar uhh... wat?
image

**Pull request merged**